src/main/java/uk/nhs/hee/tis/trainee/sync/service/PlacementSyncService.java

```diff
@@ -54,12 +58,19 @@ public void syncRecord(Record record) {
     }
   }
 
+
+
   public Set<Placement> findByPostId(String postId) {
     return repository.findByPostId(postId);
   }
 
   public void request(String id) {
-    // TODO: Implement.
-    throw new UnsupportedOperationException();
+    log.info("Sending request for Post [{}]", id);
+    String table = "Post";
```

Comment:
I think table should be `"Placement"`.  I'd refactor this to put the literal on line 71 `dataRequestService.sendMessage("Placement", id);`.